cli/tests/conftest.py

```diff
@@ -19,6 +19,14 @@ def clear_env():
         del os.environ["AWS_DEFAULT_REGION"]
 
 
+@pytest.fixture(autouse=True)
+def mock_default_instance(mocker, request):
+    # Mock get_default_instance_type for all tests.
+    if "nomockdefaultinstance" in request.keywords:
```

Comment:
Could you breefly explain this skip logic in a comment?

Reply:
Done